src/main/java/projectzero/cli/CliApplication.java

```diff
@@ -1,122 +1,351 @@
+                 System.out.println("Invalid method input");
+             }catch (NullPointerException e){
+                 System.out.println("Class not found");
+             }
+
+
+    }
+
+    private void printHelp() {
+        File help = new File("src/main/java/projectzero/cli/helpMenu");
+        try{
+            Scanner fileRead = new Scanner(help);
+            while(fileRead.hasNext()){
+                System.out.println(fileRead.nextLine());
+            }
+        }catch(Exception e){
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void addClass(String name){
+
+        try{
+            MainManager.addUmlClass(new UmlClass(name));
+            System.out.println(name + " " +  "was added");
+        }catch (InvalidNameException e){
+                System.out.println("Not a valid class name.");
+            }
+        catch (NullPointerException e){
+            System.out.println("Class already exists.");
+        }
+
+
     }
 
     private void printList() {
         List<UmlClass> tempList = MainManager.listUmlClasses();
         System.out.println("\nCurrent classes:");
-        System.out.print("[ ");
         for (UmlClass tempClass : tempList) {
-            System.out.print(tempClass.getName() + " ");
+            System.out.print("[ ");
+            System.out.println(tempClass.getName() + ": ");
+            System.out.print("\t");
+            System.out.println("Fields: ");
+            for(Field f: tempClass.getFields()){
+                System.out.println("\t  " + f.getName());
+            }
+            System.out.print("\t");
+            System.out.println("Methods: ");
+            for(Method m: tempClass.getMethods()){
+                System.out.println("\t  " + m.getName());
+            }
+            System.out.print("\t");
+            System.out.println("Relationship Points to: ");
+            for(Relationship r: tempClass.getRelationships()){
+                System.out.println("\t  " + r.getTo().getName());
+            }
+            System.out.println("]\n");
         }
-        System.out.println("]\n");
     }
+
+
+    public void displayOneClass(String name){
+        UmlClass temp = MainManager.getUmlClass(name);
+        System.out.print("[ ");
+        System.out.println(temp.getName() + ": ");
+        System.out.print("\t");
+        System.out.println("Fields: ");
+            for(Field f: temp.getFields()){
+                System.out.println("\t  " + f.getName());
+            }
+            System.out.print("\t");
+            System.out.println("Methods: ");
+            for(Method m : temp.getMethods()){
+                System.out.println("\t  " + m.getName());
+            }
+        System.out.print("\t");
+        System.out.println("Relationship Points to: ");
+        for(Relationship r: temp.getRelationships()){
+            System.out.println("\t  " + r.getTo().getName());
+        }
+            System.out.println("]\n");
```

Comment:
Same as the last comment. Possibly use StringBuilder for this. Just a suggestion for later.